File: JavaFundamentals/03_Arrays/ArraysLab/src/_01DayOfWeek.java
import java.util.Scanner;

public class _01DayOfWeek {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int dayOfWeek = scanner.nextInt();

        String[] weekDays = new String[]
                {"Monday",
                "Tuesday",
                "Wednesday",
                "Thursday",
                "Friday",
                "Saturday",
                "Sunday"};

        if (dayOfWeek >= 1 && dayOfWeek <= 7) {
            System.out.println(weekDays[dayOfWeek - 1]);
        } else {
            System.out.println("Invalid day!");
        }
    }
}

File: JavaScriptBasics/03_1_ConditionalStatementsAdvancedLab/01_DayOfWeek.js
function printDayOfTheWeek(input) {
    let weekDayNumber = input[0];
    let weekDay = "";

    switch (weekDayNumber) {
        case "1": weekDay = "Monday"; break;
        case "2": weekDay = "Tuesday"; break;
        case "3": weekDay = "Wednesday"; break;
        case "4": weekDay = "Thursday"; break;
        case "5": weekDay = "Friday"; break;
        case "6": weekDay = "Saturday"; break;
        case "7": weekDay = "Sunday"; break;
        default: weekDay = "Error"; break;
    }

    console.log(weekDay);
}

printDayOfTheWeek(["3"]);

